### ISSUE TITLE
Documentation required 

### ISSUE THREAD

**Issue by hariharan1412** [zrporz/AutoSeg-SAM2#4]
Hi, I really love your work. Is there any document where I can refer to how this project/pipeline works and what are the parameters that need tweaking? 

My project involves Equirectangluar image, so I feel like I need to tweak this mask thresold and all to get better tracking

**Comment by zrporz** [comment]
Thank you very much for your attention, in `auto-mask-batch.py` I defined some parameters that can be adjusted, you can also refer further to https://github.com/facebookresearch/segment-anything-2 and https://github.com/zrporz/segment-anything-1  You can also refer to the documentation at  and  to tweak them further.

**Comment by hariharan1412** [comment]
Yeah, I understood that now I sampled my data down to only 20 frames and itself taking more memory and time to compute. If you can explain the flow of this auto-segment in a few words, then I can also try to improve it.

Please note, I'm asking you the flow because it wasn't mentioned anywhere in your repo.
thank you

**Comment by zrporz** [comment]
Hello, our pipeline can be understood as a linear exploration of new objects at a fixed time step, specifically, given a detection step t, for the current frame T, we will use Seg2 for object tracking segmentation based on the segmentation result of Seg1, and at the moment $T+kt (k=1,2,3\cdots)$, we will detect the ratio of the currently segmented mask of the frame to the image $r_ {T+kt}$ compared to T moment $r_{T}$ whether to decline, if there is a significant decrease, can be considered to have a new object, at this time we will re-segment the scene, and according to the new segmentation results and the intersection of the previous segmentation results of the size of the intersection of the new segmentation results and determine which segmentation for the new object, and added to the current static segmentation results, and then Seg2 will be based on the static segmentation of segmentation
For performance optimization, you can refer to https://github.com/zrporz/AutoSeg-SAM2/issues/5
